.github/scripts/prebuild-case-optimization.sh

```
@@ -0,0 +1,27 @@
+#!/bin/bash
+
+# Pre-builds all benchmark cases with --case-optimization on the login node.
+# Called by the case-optimization CI job before SLURM submission.
+# Usage: bash .github/scripts/prebuild-case-optimization.sh <cluster> <device> <interface>
+
+set -e
+
+cluster=$1
+job_device=$2
```

Shellcheck (0.11.0)

[warning] job_device appears unused. Verify use (or export if used externally).

(SC2034)